projects/ccd-case-ui-toolkit/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@hmcts/ccd-case-ui-toolkit",
-  "version": "6.13.10-rc1",
+  "version": "6.10.12-judicial-user-typeahead-fix",
```

Comment:
why has the version been reduced

Reply:
@GaryScottHMCTS - thank you for pointing this out, I have now updated the version.